src/templates/all/.make/cspell.mk

```diff
@@ -12,6 +13,10 @@
 	@echo "                      to add remaining words to the project's cspell ignore list"
 
 cspell-test: docker-pull
+ifeq ("$(wildcard $(CSPELL_PROJECT_WORDS))","")
+	@echo "$(YELLOW)No $(CSPELL_PROJECT_WORDS) found, creating...$(SGR0)"
+	echo -e "\n" > $(CSPELL_PROJECT_WORDS)
```

Comment:
maybe just `touch $(CSPELL_PROJECT_WORDS)` to create an empty words file

Reply:
https://github.com/Arrow-air/tf-github/pull/42/commits/64ab221b7b24866f3107ad085e31e76fa8449bb1